assets/schema/ensemble_schema.json

```diff
@@ -808,6 +808,38 @@
         }
       }
     },
+    "LoadingContainer-payload": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "A unique identifier for this widget"
+        },
+        "isLoading": {
+          "type": "boolean",
+          "description": "Whether to display loading widget"
+        },
+        "useShimmer": {
+          "type": "boolean",
+          "description": "Whether to use a shimmer widget in loading state"
```

Comment:
What is the relation of shimmer widget to loading widget?

Reply:
It wraps the loading widget with the Shimmer widget (https://pub.dev/packages/shimmer) if `useShimmer = true`